src/cli/worker-instance.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * Worker thread that runs an instance of a mocked local fulfillment app.
+ */
+import {parentPort, workerData} from 'worker_threads';
+import {smarthomeStub} from '../platform/stub-setup';
+import {AppStub} from '../platform/smart-home-app';
+import * as fs from 'fs';
+import {PlatformWorker} from './platform-worker';
+import {READY_FOR_MESSAGE, CHECK_READY} from './commands';
+
+/**
+ * Check that this worker thread was started properly
+ */
+if (parentPort === null) {
+  throw new Error("Couldn't start worker thread: parentPort was null");
+}
+
+/**
+ * Override the constructor to capture the AppStub instance.
+ */
+let appStubInstance: AppStub | undefined = undefined;
+class CliAppStub extends AppStub {
+  constructor(version: string) {
+    super(version);
+    appStubInstance = this;
+  }
+}
+
+/**
+ * Inject the stubs into the worker thread context.
+ */
+smarthomeStub.App = CliAppStub;
+(global as any).smarthome = smarthomeStub;
+
+/**
+ * Strips the file extension off the filepath and validates it.
+ */
+const modulePath = workerData.substring(0, workerData.lastIndexOf('.'));
+if (!fs.existsSync(workerData)) {
+  throw new Error('File at path ' + workerData + ' not found.');
```

Comment:
You can use string templating here: \``File at path ${workerData} not found\``

But with one backtick on each side